cmake/projects/Boost/wave/hunter.cmake

```diff
@@ -18,5 +18,5 @@ hunter_download(
     Boost
     PACKAGE_COMPONENT
     wave
-    PACKAGE_INTERNAL_DEPS_ID "26"
+    PACKAGE_INTERNAL_DEPS_ID "27"
```

Comment:
Actually there are no changes in internal build instructions so update not needed.

Reply:
okay,  should i set it back to 26?